client/js/table-view.js

```diff
@@ -92,6 +110,22 @@ class TableView {
 		this.currentCellLocation = { col: col, row: row };
 		this.renderTableBody();
 		this.renderFormulaBar();
+		this.renderTableFooter();
+	}
+
+	sumColumnTotal(col) {
+		var total = 0;
+		// loop through each row
+		for(let row = 0; row < this.model.numRows; row++) {
+			// account for values in row 
+			if(this.model.getValue({col: col, row: row}) !== undefined) {
```

Comment:
Do you see the duplication between line 121 and 123?  Please find a way to remove it. Keep it DRY! :)